azuki-core/build.gradle

```diff
@@ -8,10 +8,13 @@ dependencies {
     implementation 'com.fasterxml.jackson.module:jackson-module-kotlin:2.12.5'
     implementation 'org.slf4j:slf4j-api:1.7.36'
 
-    runtimeOnly "org.jetbrains.kotlin:kotlin-script-runtime"
-    runtimeOnly "org.jetbrains.kotlin:kotlin-compiler-embeddable"
-    runtimeOnly "org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable"
-    runtimeOnly "org.jetbrains.kotlin:kotlin-script-util"
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-script-runtime'
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-compiler-embeddable'
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable'
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-scripting-common'
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-scripting-jvm'
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-scripting-dependencies'
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-scripting-jsr223'
```

Comment:
⛏️ There is some redundancy here, in that `kotlin-scripting-jsr223` depends on `kotlin-scripting-jvm`, `kotlin-scripting-common`, `kotlin-scripting-compiler-embeddable` and `kotlin-compiler-embeddable`, and the latter depends on `kotlin-script-runtime`.
https://central.sonatype.com/artifact/org.jetbrains.kotlin/kotlin-scripting-jsr223/1.9.0/dependencies

Anyway, if this works I don't mind.